### ISSUE TITLE
Changing status FrontEnd stopped working

### ISSUE THREAD

**Issue by StuartMcD** [solspace/craft3-freeform#1075]
Am using FF status changes as a workflow. Aside from growing pains and up until somewhere  in 10,11,12 the ability to change the status (front end) has stopped working. The form is submitting fine and on this form, notifications are still firing.

```
{% set os = craft.app.request.getParam('os') %} <!-- This corresponds with a status offer/reject -->
{% set form = freeform.form(playerSignup, {
	status: os,
	submissionToken: submissionToken|default(null),
	attributes: {
		class:"acceptForm stepped x",
		id:"discount",
	 },
	disable: true,
	})
%}

{{ form.renderTag({
	disable: ["submitButtons"],
}) }}
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.7.3
- Freeform Version: 5.0.12
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade from 5.0.10

**Comment by kjmartens** [comment]
Sorry for the trouble @StuartMcD.

I can duplicate this as well. We'll get it fixed shortly. 🙂

**Comment by kjmartens** [comment]
This is resolved now in Freeform 5.0.14. 🙂
